detd/devices/intel_i225.py

```diff
@@ -52,8 +55,6 @@ def __init__(self, pci_id):
         if pci_id in IntelI225.PCI_IDS_UNPROGRAMMED:
             raise "The flash image in this i225 device is empty, or the NVM configuration loading failed."
 
```

Comment:
Please add the capabilities in the plugin. i225, like i226, supports Qbv, Qbu and LTC.